ts/components/conversation/right-panel/overlay/disappearing-messages/TimeOptions.tsx

```diff
@@ -0,0 +1,44 @@
+import { isEmpty } from 'lodash';
+import React from 'react';
+import { TimerOptionsArray } from '../../../../../state/ducks/timerOptions';
+import { PanelButtonGroup, PanelLabel } from '../../../../buttons/PanelButton';
+import { PanelRadioButton } from '../../../../buttons/PanelRadioButton';
+
+type TimerOptionsProps = {
+  options: TimerOptionsArray | null;
+  selected?: number;
+  setSelected: (value: number) => void;
+  hasOnlyOneMode?: boolean;
+  disabled?: boolean;
+};
+
+export const TimeOptions = (props: TimerOptionsProps) => {
+  const { options, selected, setSelected, hasOnlyOneMode, disabled } = props;
+
+  if (!options || isEmpty(options)) {
```

Comment:
I don't think I can because typescript will give an error.

Reply:
right, stupid typescript

Reply:
Turns out we need to update our lodash type definitions. This will resolve this issue but we need to update a few cases so will be done as a separate PR.